kubernetes/prod/08-easyshop-deployment.yaml

```
@@ -0,0 +1,128 @@
+            - name: cache-volume
+              mountPath: /app/.next/cache
+      volumes:
+        - name: tmp-volume
+          emptyDir: {}
+        - name: cache-volume
+          emptyDir:
+            sizeLimit: 1Gi
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)